pkg/NAMESPACE

```diff
@@ -132,6 +134,9 @@ exportMethods("asc",
               "countDistinct",
               "sumDistinct")
 
+exportClasses("GroupedData")
+exportMethods("count", "agg", "avg")
```

Comment:
`count` and `avg` definitely are, though I think `agg` still needs to be exported.